hack/start-api.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# password hash is 'admin'
+
+ADMIN_ACCOUNT_ENABLED=true \
+ADMIN_ACCOUNT_TOKEN_AUDIENCE=kargo.demo.akuity.io \
+ADMIN_ACCOUNT_TOKEN_ISSUER=http://kargo.demo.akuity.io \
```

Comment:
```suggestion
ADMIN_ACCOUNT_TOKEN_AUDIENCE=localhost \
ADMIN_ACCOUNT_TOKEN_ISSUER=localhost \
```

I see that this is set to `kargo.demo.akuity.io`, which is our demo instance URL. I'm pretty sure this doesn't matter since I believe these values are arbitrary. Suggest updating it to `localhost`